src/multi_vector_simulator/F0_output.py

```diff
@@ -229,3 +237,41 @@ def store_as_json(dict_values, output_folder=None, file_name=None):
         answer = json_data
 
     return answer
+
+
+def select_essential_results(dict_values):
+    """
+
+    Parameters
+    ----------
+    dict_values : (dict)
+        dict to be stored as json
+
+    Returns
+    -------
+    None
+    Remove certain fields in dict_values
+
+    """
+
+    # Remove the input data which where not modified
+    for asset_group in [
+        ECONOMIC_DATA,
+        SIMULATION_SETTINGS,
+        FIX_COST,
+        ENERGY_BUSSES,
+        PATHS_TO_PLOTS,
+        OPTIMIZED_FLOWS,
```

Comment:
This may work for now, but in a different pr we should find a way to define energy busses manually

Reply:
Not sure if we will run into issues this way, as `energyBusses` bay be overwritten with `C`

Reply:
That is possible, I did not check how the output looked, I only checked that the simulation was running through with the provided energyBusses, thinking we will then iterate on EPA outputs which are then input of MVS etc...